components/block_v2/templates/block.jade

```diff
@@ -0,0 +1,20 @@
+- extraClass = block.kind.__typename === 'Channel' ? "Block__inner--channel--" + block.kind.visibility : ''
```

Comment:
I don't get this whole `extraClass` thing. Name it something meaningful or inline it or extract it to a helper.